src/main/java/org/isf/menu/manager/UserBrowsingManager.java

```diff
@@ -90,9 +87,39 @@ public User getUserByName(String userName) throws OHServiceException {
 		return ioOperations.getUserByName(userName);
 	}
 
+	/**
+	 * Returns {@link User} from its username
+	 * @param username - the {@link User}'s username
+	 * @param withTrashed - Included soft deleted if set to true
+	 * @return {@link User}
+	 * @throws OHServiceException When error occurs
+	 */
+	public User getUserByName(String username, boolean withTrashed) throws OHServiceException {
```

Comment:
I would vote to change `withTrashed` to something like `withSoftDeletion`.    `trashed` doesn't really lend itself to both soft and hard deletes IMHO.

Reply:
voted!